.builder/actions/aws_crt_python.py

```diff
@@ -48,12 +49,15 @@ def run(self, env):
         self._setenv_tmpfile_from_secret('AWS_TEST_TLS_CERT_PATH', 'unit-test/certificate', 'certificate.pem')
         self._setenv_tmpfile_from_secret('AWS_TEST_TLS_KEY_PATH', 'unit-test/privatekey', 'privatekey.pem')
 
+        self._setenv_tmpfile_from_secret('AWS_TEST_ECC_CERT_PATH', 'ecc-test/certificate', 'ECCcertificate.pem')
+        self._setenv_tmpfile_from_secret('AWS_TEST_ECC_KEY_PATH', 'ecc-test/privatekey', 'ECCprivatekey.pem')
+
         # enable S3 tests
         env.shell.setenv('AWS_TEST_S3', '1')
 
         self._try_setup_pkcs11()
 
-    def _get_secret(self, secret_id):
+    def _get_secret(self, secret_id, binary=False):
```

Comment:
it looks like you added `binary` argument but then never used it?

If you don't need it, revert all the changes related to binary stuff, less code is best code